### PR TITLE
fix fullscreen a11y issues

### PR DESCRIPTION
**Pull request by talevy17**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,6 @@
     "build:dev": "lerna run --stream --ignore 'wix-rich-content-@(test-env|example|storybook)' build:dev",
     "build:core": "npm run build:dev -- --scope 'ricos-content' && npm run build:dev -- --scope 'wix-rich-content-@(common|editor|viewer)'",
     "prewatch": "lerna run build --scope wix-rich-content-*common* && lerna run build --scope wix-rich-content-editor && lerna run build --scope ricos-common",
-    "prebuild": "node scripts/mergeTranslations",
```
Comment (maimonav) [inline package.json:33]: It is on purpose?

File: packages/fullScreen/web/src/InnerFullScreen.jsx [hunk 2]
```diff
@@ -101,7 +112,7 @@ export default class InnerFullscreen extends Component {
         tabIndex={0}
         className={styles.close}
         onClick={this.onClose}
-        onKeyDown={this.onClose}
+        onKeyDown={e => e.keyCode === 13 && this.onClose()}
```
Comment (maimonav) [inline packages/fullScreen/web/src/InnerFullScreen.jsx:115]: I suggest to extract keyCode === 13 to a method

File: packages/fullScreen/web/src/InnerFullScreen.jsx [hunk 3]
```diff
@@ -113,7 +124,7 @@ export default class InnerFullscreen extends Component {
   onFullscreenToggleKeyDown = event => {
     if (event.key === 'Escape') {
       this.onClose();
-    } else {
+    } else if (event.keyCode === 13) {
```
Comment (maimonav) [inline packages/fullScreen/web/src/InnerFullScreen.jsx:127]: Check comment in line 115. :)

File: package.json (hunk 1 above)
Comment (talevy17) [reply package.json:33]: mistake reverting

File: packages/fullScreen/web/src/InnerFullScreen.jsx [hunk 4]
```diff
@@ -121,14 +123,6 @@ export default class InnerFullscreen extends Component {
     );
   };
 
-  onFullscreenToggleKeyDown = event => {
-    if (event.key === 'Escape') {
```
Comment (maimonav) [inline packages/fullScreen/web/src/InnerFullScreen.jsx:114]: Isn't this use case is missing now?

Comment (talevy17) [reply packages/fullScreen/web/src/InnerFullScreen.jsx:114]: @maimonav nope, its already a registered event listener in `document` and works either way.

**Review by maimonav — APPROVED** [review]

**Pull request merged**